packages/frontend/app/routes/_index.tsx

```diff
@@ -23,6 +93,16 @@ export default function Index() {
             In this web application, you&apos;ll be able to manage peering
             relationships, assets, and wallet addresses, among other settings.
           </p>
+
+          <div className='space-y-4'>
+            <p className='text-gray-600'>
+              To get started, please configure your API credentials
+            </p>
+            <div className='max-w-md mx-auto'>
+              <ApiCredentialsForm hasCredentials={hasCredentials} />
```

Comment:
New form to submit credentials. 

I included it on the homepage because thats naturally where you land and you cant really see anything else without submitting these.

Form validates uuid client side. Saves to cookie based session on submit which you can also delete. Here are the views for credentials set in storage and not set.

![image](https://github.com/user-attachments/assets/98ca5353-385b-428c-8856-79a80d526daf)

![image](https://github.com/user-attachments/assets/e370f3fe-7ea9-467e-860e-09610bbe3908)